.github/workflows/build-and-push-image.yml

```diff
@@ -12,15 +12,15 @@ on:
         required: true
         type: string
       gar_location:
-        required: true
+        required: false
```

Comment:
What happens if these are not specified? I see that the tags below still draw from `env.GAR_LOCATION` for instance. I presume that those are just being inherited?

Reply:
I encountered issues when I required inputs even with defaults supplied here: https://github.com/mozilla-services/autopush-rs/actions/runs/10605118999

I figured it would make the most sense to specify the defaults here instead of including them as inputs upstream in docker.yml.

In this case, since I specify default values when `required: false` is set, we should be fine. This particular workflow may only be triggered by another workflow, so there should be no chance we see any errors.

Reply:
Ah, cool. We probably ought to document that somewhere so that future us don't do dumb things, but that can be a different PR.

Reply:
I don't mind throwing in a comment now!

Reply:
Added a comment to that file now